karma.conf.js

```diff
@@ -173,5 +173,94 @@ module.exports = function (config) {
     options.detectBrowsers.enabled = false;
     options.browsers = process.env.BROWSERS.split(" ");
   }
+
+  if (process.env.GITHUB_WORKFLOW) {
+    const { readdirSync } = require("fs");
+    const defaultPlugins = readdirSync("node_modules").filter(m =>
+      m.startsWith("karma-")
```

Comment:
Didn't `"karma-*"` work? 😮

Reply:
Oh. You literally meant `"karma-*"`. :sweat_smile: